attributes/default.rb

```diff
@@ -1,40 +1,74 @@
-# Ice version
+# Ice version to download and install.  These versions are packaged and hosted
+# by Medidata Solutions until we can get the Netflix Ice team to package and
+# host official ice releases.  If you wish to install the latest stable version
+# from [Netflix](https://github.com/netflix/ice#download-snapshot-builds)
+# directly, provide `stable`. Note: this option will always install the latest
+# version even if its not backwards compatible.
 node.default['ice']['version']                              = '0.0.4'
+
+# Checksum for Ice WAR file.
 node.default['ice']['checksum']                             = 'eb9e7503585553bdebf9d93016bcbe7dc033c21e2b1b2f0df0978ca2968df047'
+
+# HTTP URL for Ice WAR file.
 node.default['ice']['war_url']                              = 'https://s3.amazonaws.com/dl.imedidata.net/ice'
+
+# Will force a redeploy of the Ice WAR file.
 node.default['ice']['force_redeploy']                       = false
 
+# Organization name that is displayed in the UI header.
 node.default['ice']['company_name']                         = nil
 
+# Enables the ice processor.
 node.default['ice']['processor']['enabled']                 = true
-node.default['ice']['processor']['local_dir']                = '/var/ice_processor'
+
+# Local work directory for the Ice processor.
+node.default['ice']['processor']['local_dir']               = '/var/ice_processor'
+
+# Optional. Work around https://github.com/Netflix/ice/issues/100 by pre-creating
+# the directories it expects to find. Default: false
 node.default['ice']['processor']['issue_100_workaround']    = false
 
-# S3 keys for accessing billing files
+# AWS access key id used for accessing AWS detailed billing files from S3.
 node.default['ice']['billing_aws_access_key_id']            = nil
+# AWS secret key used for accessing AWS detailed billing files from S3.
 node.default['ice']['billing_aws_secret_key']               = nil
```

Comment:
These values are not seperated by a line, when most other values are. I know they are related, just stylistic choice that is up to you.